.circleci/config.yml

```
@@ -537,6 +540,33 @@ jobs:
       - store_artifacts:
           path: test-artifacts
           destination: test-artifacts
+    
+  test-e2e-chrome-mv3:
+    executor: node-browsers
+    steps:
+      - checkout
+      - run:
+          name: Re-Install Chrome
+          command: ./.circleci/scripts/chrome-install.sh
+      - attach_workspace:
+          at: .
+      - run:
+          name: Move test build to dist
+          command: mv ./dist-test-mv3 ./dist
+      - run:
+          name: Move test zips to builds
+          command: mv ./builds-test-mv3 ./builds
+      - run:
+          name: test:e2e:chrome
+          command: |
+            if .circleci/scripts/test-run-e2e.sh
+            then
+              yarn test:e2e:chrome --retries 2
```

Even if we don't make this status check required, this will still put a red "X" by each PR. Not ideal.

We should be able to suppress that here by handling the failure case:

```suggestion
              yarn test:e2e:chrome --retries 2 || echo "Temporarily suppressing MV3 e2e test failures"
```

---

that's a great suggestion!! thank you so much @Gudahtt